translate/eslint.config.mjs

```diff
@@ -0,0 +1,49 @@
+import typescriptEslint from '@typescript-eslint/eslint-plugin';
+import _import from 'eslint-plugin-import';
+import { fixupPluginRules } from '@eslint/compat';
+import globals from 'globals';
+import tsParser from '@typescript-eslint/parser';
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+
+export default [
+  eslint.configs.recommended,
+  ...tseslint.configs.recommended,
+  {
+    plugins: {
+      '@typescript-eslint': typescriptEslint,
+      import: fixupPluginRules(_import),
+    },
+
+    languageOptions: {
+      globals: {
+        ...globals.browser,
+        ...globals.jest,
+      },
+
+      parser: tsParser,
+    },
+
+    rules: {
+      curly: 'error',
+      'prefer-const': 0,
+      'no-var': 0,
+      '@typescript-eslint/ban-ts-comment': 0,
+      '@typescript-eslint/ban-types': 0,
+      '@typescript-eslint/explicit-module-boundary-types': 0,
+      '@typescript-eslint/no-empty-function': 0,
+      '@typescript-eslint/no-explicit-any': 0,
+      '@typescript-eslint/no-inferrable-types': 0,
+
+      '@typescript-eslint/no-unused-vars': [
+        'error',
+        {
+          varsIgnorePattern: '^_',
+        },
+      ],
+
+      '@typescript-eslint/prefer-as-const': 0,
+      'import/no-default-export': 'error',
```

Comment:
This doesn't need a plugin: https://eslint.org/docs/latest/rules/no-restricted-exports#restrictdefaultexports